server/accounts.go

```diff
@@ -3544,21 +3600,26 @@ func (s *Server) fetch(res AccountResolver, name string) (string, error) {
 	return theJWT, err
 }
 
-func NewCacheDirAccResolver(path string, limit int64, ttl time.Duration) (*CacheDirAccResolver, error) {
+func NewCacheDirAccResolver(path string, limit int64, ttl time.Duration, _ ...interface{}) (*CacheDirAccResolver, error) {
```

Comment:
No, this is not I had in mind. As explained on Slack, if you want to make it extensible in the future, you should replace all params with a public option structure, or a variadic of options. If you don't plan to add anything after the public release, then remove and do nothing.

Reply:
Should you remove from this one? Is this a jwt store? If not, then you should not be using that, otherwise, it should be `_ ...dirJWTStoreOption` with your current approach.